Pipfile

```diff
@@ -21,6 +21,7 @@ thoth-analyzer = "*"
 thoth-common = "*"
 thoth-python = "*"
 thoth-storages = "*"
+flask-cors = "*"
```

Comment:
One minor thing, you will need to run:

```pipenv lock``` and commit also Pipfile.lock

Reply:
sir i have been trying to run it. But every time it gets stuck while locking dependencies. I have left it running for hours.

Reply:
It took few minutes on my system (5-7 mins, using Fedora:30):

```
$ pipenv --version                                                                                                                                                                                                            
pipenv, version 2018.11.26
```

Reply:
Sir should i use exact same pipenv version. I am on Ubuntu 18.04.

Reply:
You don't need to, I posted the Pipenv version in case you have some troubles to verify version I'm running.

Reply:
Sir it is working now. From what I understood the issue previously was that I was trying to do pip lock without using the `flask_cors` library in my code.